### PR TITLE
Map ilike to LIKE

### PR DESCRIPTION
**Pull request by Soulforged**
Solves https://github.com/livehelpnow/tds_ecto/issues/24

### CODE REVIEW

**Comment by Soulforged** [comment]
Fixes https://github.com/livehelpnow/tds_ecto/issues/26 and https://github.com/livehelpnow/tds_ecto/issues/24

File: mix.exs [hunk 1]
```diff
@@ -23,7 +23,7 @@ defmodule TDS.Ecto.Mixfile do
     [
       {:ecto, "~> 1.0.0"},
       {:tds, "~> 0.5.1"},
-      {:poison, only: :test}
+      {:poison, "~> 1.0"}
```
Comment (mobileoverlord) [inline mix.exs]: How come we are requiring poison outside of test?

Comment (Soulforged) [inline mix.exs]: Yes, the thing is that I'm using the latest mix version, and if I use a more restrictive dependency specification it generates a conflict with other packages that require poison as optional but not test only. I can remove that change though, it's not important.

Comment (mobileoverlord) [inline mix.exs]: Yeah, lets not require Poison, we can clean it up later to operate better with mix. I'll merge this in then and release a new version. Thanks for the PR!!! :bowtie:

Comment (Soulforged) [inline mix.exs]: :+1:

**Comment by mobileoverlord** [comment]
@Soulforged Sorry for the delay, is this good to go?

**Comment by Soulforged** [comment]
Yep, it should.

**Comment by mobileoverlord** [comment]
:) Thanks Ill test it out and create a new release.

**Pull request merged**